carbon-design-system/carbon-addons-iot-react
# feat(table): added column overflow menu

Closes #1375, and #1597 

**Summary**

- Adds support for an overflow menu in the the column header

**Change List (commits, features, bugs, etc)**

- Added column header and callback for both table and stateful table

**Acceptance Test (how to verify the PR)**

- Check out the `Simple Stateful Example with column overflow menu` story and click the overflow item + options. Observe the storybook actions tab for callbacks.

- Check column size adjust and column resort to make sure those are working correctly
